.github/workflows/breakage.yml

```diff
@@ -0,0 +1,65 @@
+name: Breakage
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  break:
+    name: Breakage of ${{ matrix.pkg }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        pkg: [AmplNLReader, CUTEst, CaNNOLeS, NLPModelsIpopt, NLPModelsJuMP, QuadraticModels, SolverTools]
```

Comment:
Why is NLPModelsIpopt in here again?

Reply:
This is part of the original selection of packages we were testing for breakage. It's here because it uses NLPModels.